contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_line_km_distance_between.py

```diff
@@ -0,0 +1,182 @@
+                            ]
+                        )
+                    ),
+                    mapping(
+                        LineString(
+                            [
+                                (556597.453966, 552188.640112),
+                                (1224514.398726, 1209055.279421),
+                                (779236.435553, 881798.964757),
+                            ]
+                        )
+                    ),
+                ],
+            },
+            "tests": [
+                {
+                    "title": "basic_positive_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "linestring_less_than_1000_km",
+                        "min_distance": 0,
+                        "max_distance": 1000,
+                    },
+                    "out": {
+                        "success": True,
+                    },
+                },
+                {
+                    "title": "basic_negative_test",
+                    "exact_match_out": False,
+                    "include_in_gallery": True,
+                    "in": {
+                        "column": "linestring_between_1000_and_2000_km",
+                        "min_distance": 1000,
+                        "max_distance": 2000,
+                    },
+                    "out": {
+                        "success": False,
+                    },
+                },
+            ],
+        }
+    ]
+
+    # This dictionary contains metadata for display in the public gallery
+    library_metadata = {
+        "maturity": "experimental",  # "experimental", "beta", or "production"
+        "tags": [
+            "geospatial",
+            "hackathon-22",
+        ],  # Tags for this Expectation in the gallery
+        "contributors": [  # Github handles for all contributors to this Expectation.
+            "@luismdiaz01",
+            "@derekma73",
+        ],
+        "requirements": ["geopandas", "shapely"],
+    }
+
+    # This is the id string of the Metric used by this Expectation.
+    # For most Expectations, it will be the same as the `condition_metric_name` defined in your Metric class above.
+    map_metric = "column_values.linestring_distance_km"
+
+    # This is a list of parameter names that can affect whether the Expectation evaluates to True or False
+    # Please see {some doc} for more information about domain and success keys, and other arguments to Expectations
+    success_keys = (
+        "mostly",
+        "min_distance",
+        "max_distance",
+    )
+
+    # This dictionary contains default values for any parameters that should have default values
+    default_kwarg_values = {
+        "min_distance": 0,
+        "max_distance": 0,
+        "mostly": 1.0,
+    }
+
+
+if __name__ == "__main__":
+    ExpectColumnValuesToBeLineKMDistanceBetween().print_diagnostic_checklist()
```

Comment:
Linting error; Missing newline at end of file

Reply:
Thanks for that @austiezr ! Fixed the issue